### DISCUSSION TITLE
Storemanager overrides my HTML and CSS

### DISCUSSION THREAD

**Discussion by FilipZamec** [GrapesJS/grapesjs#5512] [Q&A]
Hi, I need some help. 
I'm new to GrapesJS. Great project btw.
I have searched for this in discussions, but I couldn't find the answer.

I have this simple code, which successfully loads my HTML and CSS.
```javascript
static async initEditor(html, css) {
        if (this.editor)
            this.editor.destroy();
        this.editor = grapesjs.init({
            container: '#gjs',
            components: html,
            style: css             
       });
}
```
But if I call this method for a second time with different content, nothing gets updated. I understand the content is stored in local storage.

so if I add: `storageManager : false` in the init function then everything works as expected.

My problem is this:
1. I don't understand why Storagemanager overides my components and styles. I would understand it, if I woudn't set it.
2. I want to use Symbols plugin, which needs Storagemanager for storing symbols, so I want to store symbols across pages, but I want to load different HTML and CSS every time.

How do I do that? Thanks.

**Comment by artf** [comment]
The Storage Manager [autoloads](https://github.com/GrapesJS/grapesjs/blob/8025b95a645907574389e950f32950190ceb602c/src/storage_manager/config/config.ts#L32) the content if there is something stored
